code/R/model_pipeline.R

```diff
@@ -39,7 +39,7 @@ source("code/R/tuning_grid.R")
 source("code/R/permutation_importance.R")
 
 pipeline <- function(data, model, split_number, outcome=NA, hyperparameters=NULL, permutation=TRUE){
-
+	data <- read.csv('test/data/small_input_data.csv')
```

Comment:
Thanks @pschloss! Could you remove this line before I do a test-run?

Reply:
ack. i knew i'd forget to remove that!

Reply:
Thanks @pschloss. I tested and it works.